### DISCUSSION TITLE
Resubmitting Assignments

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#36] [General]
How long do we have to resubmit old assignments to try and receive a better grade?

**Comment by cooperrc** [comment]
Hard deadline is Fri Jul 1 for all submissions. Otherwise, I can't include the assignment in the final grade.

**Reply by ecieszynski** [reply]
To resubmit an assignment after its due do we simply unsubmit the original submission then turn in the new one?

**Reply by cooperrc** [reply]
:+1: Yes

Just add a new file (like HW_01-revised) so I can keep track of progress.
